pinc/misc.inc

```diff
@@ -444,6 +444,27 @@ function javascript_safe($str, $encoding=NULL)
     return $str;
 }
 
+// -----------------------------------------------------------------------------
+
+function search_safe($str)
+// Return a string that is safe for searching.
+//
+// Example usage:
+//   echo search_safe("William  Shakespeare");
+//   echo search_safe("William Shakespeare ");
+//   echo search_safe(" William Shakespeare");
+//   echo search_safe("\tWilliam\tShakespeare\t");
+//
+//   They all return "William Shakespeare".
+//
+//
+// Replaces all occurrences of whitespace characters with a single spaces and
```

Comment:
"all occurrences" -> "each run"
"a single spaces" -> "a single space"

Reply:
Thanks, fixed in d32dca8c4f0ea152e3a4b5e9d600d49b5bf78723 :)